### PR TITLE
fix(*): clear errors on navigation and stop errors in tech record non-edit mode

### PR DESCRIPTION
**Pull request by aaron-downing**
Changes:
- Removes persistence of errors across pages, errors will now be cleared whenever user navigates through VTM 
- Errors will no longer be displayed on a tech record in non-edit mode that were previously caused by values on the tech record not matching the validations we have on the form.

### CODE REVIEW

**Review by naathanbrown — APPROVED** [review]

**Review by tom-bjss — DISMISSED** [review]

**Review by tahayazici — APPROVED** [review]

File: src/app/core/components/global-error/global-error.service.ts [hunk 1]
```diff
@@ -12,8 +13,13 @@ import { GlobalError } from './global-error.interface';
 export class GlobalErrorService {
   private errors: Observable<GlobalError[]>;
 
-  constructor(private store: Store<State>) {
+  constructor(private store: Store<State>, private router: Router) {
     this.errors = this.store.pipe(select(globalErrorState));
+    this.router.events.subscribe((event)=> {
```
Comment (gjulien-bjss) [inline src/app/core/components/global-error/global-error.service.ts:18]: Not super necessary, and I don't know if it's a better implementation, but the ngrx route store package dispatches actions when the user navigates. Maybe creating an effect which listens to these actions and clears the errors could be a cleaner/better implementation. Just a suggestion mostly
ngrx

**Review by gjulien-bjss — APPROVED** [review]

File: src/app/core/components/global-error/global-error.service.ts (hunk 1 above)
Comment (mcadenas-bjss) [reply src/app/core/components/global-error/global-error.service.ts:18]: Global errors reducer might be able to act on the `navigated` action and clear errors.

**Review by mcadenas-bjss — APPROVED** [review]
Looks good but I like @gjulien-bjss suggestion.

File: src/app/core/components/global-error/global-error.service.ts (hunk 1 above)
Comment (aaron-downing) [reply src/app/core/components/global-error/global-error.service.ts:18]: I'll look into this, it's a good suggestion

**Review by gjulien-bjss — APPROVED** [review]

**Review by tom-bjss — APPROVED** [review]

**Pull request merged**